pootle/settings/20-backends.conf

```diff
@@ -81,11 +78,25 @@ RQ_QUEUES = {
     },
 }
 
-# The directory where Pootle writes event logs to
+#
+# Logging
+#
+
+# The directory where Pootle writes its logs
 POOTLE_LOG_DIRECTORY = working_path("log")
 
+# Useful references:
+#
+# Logging configuration:
+# Django: https://docs.djangoproject.com/en/1.7/topics/logging/#configuring-logging
+# RQ: https://github.com/ui/django-rq#configuring-logging
+# Sentry: http://raven.readthedocs.org/en/latest/integrations/django.html#integration-with-logging
+#
```

Comment:
ES: http://elasticsearch-py.readthedocs.org/en/master/#logging

...and ideally Pootle should also need its own reference, but that requires reviewing existing loggers and naming them consistently.